### DISCUSSION TITLE
Welcome to qiuxiang Discussions!

### DISCUSSION THREAD

**Discussion by qiuxiang** [qiuxiang/qiuxiang#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by qiuxiang** [comment]
@weirongxu 来吹水，我测试下 GitHub 的通知

**Reply by weirongxu** [reply]
:raised_back_of_hand: 有什么好主题吗

**Reply by weirongxu** [reply]
你用的是邮箱通知，还是 github 内部的通知？

**Reply by qiuxiang** [reply]
邮箱通知感觉也不太合适，github 有发邮件给我，但我的邮箱没有设置通知

**Reply by weirongxu** [reply]
我是用邮箱通知的

**Reply by qiuxiang** [reply]
我的邮箱也是经常错过消息，感觉我要重新调校下 gmail 了，把优先级调下
